backend/benefit/helsinkibenefit/settings.py

```diff
@@ -345,6 +346,7 @@
 AZURE_ACCOUNT_NAME = env("AZURE_ACCOUNT_NAME")
 AZURE_ACCOUNT_KEY = env("AZURE_ACCOUNT_KEY")
 AZURE_CONTAINER = env("AZURE_CONTAINER")
+AZURE_URL_EXPIRATION_SECS = env("AZURE_URL_EXPIRATION_SECS")  # default 900s
```

Comment:
Having the default as a comment is unnecessary and it can go out of sync.

Reply:
fixed comment.